Website/src/rendering/src/components/NonSitecore/Accordion.tsx

```diff
@@ -0,0 +1,59 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { useState } from 'react';
+import Skeleton from 'react-loading-skeleton';
+
+type AccordionProps = {
+  heading: string;
+  items: AccordionItemProps[];
+  additionalClasses?: string;
+  loading?: boolean;
+};
+
+type AccordionItemProps = {
+  heading: string;
+  description: string;
+  disabled?: boolean;
+  loading?: boolean;
+};
+
+const Accordion = (props: AccordionProps): JSX.Element => {
+  const heading = !props.loading && props.heading;
+  const loadingClass = props.loading ? 'loading' : '';
+  const additionalClasses = props.additionalClasses ? props.additionalClasses : '';
+
+  return (
+    <div className={`accordion ${loadingClass} ${additionalClasses}`}>
+      <div className="accordion-heading">{heading}</div>
+      <div className="accordion-body">
+        {props.items.map((item, i) => {
```

Comment:
Better extract into a const to be consistent with no logic in JSX return statements